templates/default/systemd/sensu-api.service.erb

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=sensu api
+
+[Service]
+User=sensu
+Group=sensu
+EnvironmentFile=/etc/default/sensu
+ExecStart=/opt/sensu/bin/sensu-api -c /etc/sensu/config.json -d /etc/sensu/conf.d -e /etc/sensu/extensions -l /var/log/sensu/sensu-api.log
```

Comment:
@willejs take a look at the [changes committed to `release/0.27`](https://github.com/sensu/sensu-build/compare/master...release/0.27) branch of sensu/sensu-build. I think ideally automation would use the updated systemd files from the package instead of duplicating unit files in the cookbook.